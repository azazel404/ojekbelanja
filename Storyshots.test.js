import initStoryshots from "@storybook/addon-storyshots";

require("./.storybook/shim.js");
global.requestAnimationFrame(() => {});

initStoryshots();
