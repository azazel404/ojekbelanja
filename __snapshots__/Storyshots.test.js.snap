exports[`Storyshots Brand default 1`] = `
<span
  className="Brand">
  Ojek 
  <i
    aria-hidden="true"
    className="fa fa-motorcycle" />
  <i
    aria-hidden="true"
    className="fa fa-shopping-cart" />
   Belanja
</span>
`;

exports[`Storyshots Button default 1`] = `
<div
  style={
    Object {
      "width": 200,
    }
  }>
  <button
    className="Button Button-fullwidth"
    disabled={undefined}
    onClick={undefined}
    title=""
    type="button">
    <i
      aria-hidden="true"
      className="fa fa-font-awesome fa-lg" />
     Button
  </button>
</div>
`;

exports[`Storyshots Button icon 1`] = `
<div
  style={
    Object {
      "width": 200,
    }
  }>
  <button
    className="Button Button-icon"
    disabled={undefined}
    onClick={undefined}
    title="Hapus"
    type="button">
    <i
      aria-hidden="true"
      className="fa fa-trash fa-lg" />
  </button>
</div>
`;

exports[`Storyshots Button primary 1`] = `
<div
  style={
    Object {
      "width": 200,
    }
  }>
  <button
    className="Button Button-fullwidth"
    disabled={undefined}
    onClick={undefined}
    title=""
    type="button">
    <i
      aria-hidden="true"
      className="fa fa-shopping-cart fa-lg" />
     Fullwidth
  </button>
</div>
`;

exports[`Storyshots Button secondary 1`] = `
<div
  style={
    Object {
      "width": 200,
    }
  }>
  <button
    className="Button Button-content Button-secondary"
    disabled={undefined}
    onClick={undefined}
    title=""
    type="button">
    <i
      aria-hidden="true"
      className="fa fa-shopping-cart fa-lg" />
     Content
  </button>
</div>
`;
